docs/geedocs/answer/7160000.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8"/>
+<title>Release notes: Open GEE 5.2.0</title>
+<link rel="stylesheet" href="../css/styles.css" type="text/css" media="all" />
+<link rel="stylesheet" href="../css/containers.css" type="text/css" media="all" />
+</head>
+<body>
+<!-- 7160000.html -->
+<div class="container">
+<div class="sidebar1">
+  <p class="sidebar-toc"><a href="../answer/7160000.html" class="current-file">Open GEE 5.2.0</a></p>
+  <p class="sidebar-toc"><a href="../answer/7159245.html">GEE 5.1.3</a></p>
+  <p class="sidebar-toc"><a href="../answer/6259403.html">GEE 5.1.2</a></p>
+  <p class="sidebar-toc"><a href="../answer/6159443.html">GEE 5.1.1</a></p>
+  <p class="sidebar-toc"><a href="../answer/6078762.html">GEE 5.1.0</a></p>
+  <p class="sidebar-toc"><a href="../answer/6053170.html">GEE 5.0.2</a></p>
+  <p class="sidebar-toc"><a href="../answer/4556704.html">GEE 5.0.1</a></p>
+  <p class="sidebar-toc"><a href="../answer/3424202.html">GEE 5.0</a></p>
+</div>
+<div class="content">
+<a name="top_of_file"></a>
+<p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
+<h1><a href="../index.html">Google Earth Enterprise Documentation Home</a> | Release notes</h1>
+<h2>Release notes: Open GEE 5.2.0</h2>
+<p>Open GEE 5.2.0 the first open source release of Google Earth Enterprise.  It is designed to be compatible with GEE 5.1.3 with the exception of MrSID support.
+Open GEE 5.2.0 includes a number of code and library changes to support the open source effort.</p>
+  <h5>
+    <p id="overview_5">New Features</p>
+  </h5>
+    <p><strong>Open Sourcing</strong>. Open GEE 5.2.0 includes a number of changes that were necessary for the open sourcing effort.  This includes several library replacements, library unbundling,
+removal of obsolete code and libraries, changes to support new compilers, updates to support git and Github, and several documentation changes.</p>
+    <p><strong>New Installers</strong>. Previous GEE installers used proprietary tools that are not appropriate for open source projects.  Open GEE 5.2.0 includes new installers that run as bash
+scripts.  There are also new build and install scripts for the Portable Server.</p>
+    <p><strong>New Tutorial Files</strong>. Images used in the Fusion tutorial are now provided in both tif and jp2 format.  In addition, the tutorial files are stored and downloaded separately
+from the rest of the Open GEE code.</p>
+    <p><strong>OpenGEE.org Web Site</strong>. The Open GEE repository now includes the source code for the Open GEE website, which can be viewed at <a href="http://www.opengee.org">http://www.opengee.org</a>.</p>
+    <p><strong>New --listvolumes Flag in geconfigureassetroot</strong>. The geconfigureassetroot utility now supports a --listvolumes flag that allows users to display the configured volumes for the
+current asset root without stopping Fusion.</p>
+  <h5>
```

Comment:
I think it would be worth mentioning: https://github.com/google/earthenterprise/issues/196

It removed redundancy/confusion with configuring SSL and enabled TLS 1.2 by default.